nym-vpn-core/crates/nym-vpn-lib/src/tunnel_state_machine/states/disconnected_state.rs

```diff
@@ -12,7 +12,7 @@ use crate::tunnel_state_machine::{
     NextTunnelState, PrivateTunnelState, SharedState, TunnelCommand, TunnelStateHandler,
 };
 
-pub struct DisconnectedState;
```

Comment:
Why?

Reply:
An incomplete revert from an intermediate commit. I went back both here and at `ErrorState`